src/auxkernels/SATurbulentViscosityAux.C

```diff
@@ -0,0 +1,34 @@
+#include "SATurbulentViscosityAux.h"
+#include "metaphysicl/raw_type.h"
+
```

Comment:
MOOSE does not have a standard practice regarding `std`. For me personally, I would rather avoid potential name collisions because MOOSE overloads some math functions.